basket/news/tasks.py

```diff
@@ -201,6 +203,45 @@ def gmttime():
 
 
 @et_task
+def fxa_delete(data):
+    sfmc.upsert_row('FXA_Deleted', {'FXA_ID': data['uid']})
+
+
+@et_task
+def fxa_verified(data):
+    """Add new FxA users to an SFMC data extension"""
+    # used to be handled by the fxa_register view
+    email = data['email']
+    fxa_id = data['uid']
+    locale = data['locale']
+    subscribe = data.get('marketingOptIn')
+    metrics = data.get('metricsContext', {})
+
+    # if we're not using the sandbox ignore testing domains
+    if email_is_testing(email):
+        return
+
+    lang = get_best_language(get_accept_languages(locale))
+    if not lang:
+        return
+
+    _update_fxa_info(email, lang, fxa_id)
+
+    if subscribe:
```

Comment:
@rfk I guess we'll need to implement this bit as well, and add those new campaigns from the bug?

https://github.com/mozilla/fxa-basket-proxy/blob/13ed0f5206f9150566135d3a0973ac358cf1c0f7/lib/events/index.js#L96

Reply:
Actually nevermind. Won't need to do that until we takeover the "login" events in #37. I'll add a note there to be sure to do this bit.